R/mergelist.R

```diff
@@ -143,6 +143,120 @@ dtmerge = function(x, i, on, how, mult, join.many, void=FALSE, verbose) {
   list(ans=ans, irows=irows, xrows=xrows)
 }
 
+# atomic join between two tables
+mergepair = function(lhs, rhs, on, how, mult, lhs.cols=names(lhs), rhs.cols=names(rhs), copy=TRUE, join.many=TRUE, verbose=FALSE) {
```

Comment:
@jangorecki can `merge.data.table()` just use `mergepair()` in its implementation?

Reply:
Yes, definitely.